pyproject.toml

```diff
@@ -29,7 +29,7 @@ classifiers = [
 dependencies = [
   'colorama; os_name == "nt"',
   'meson>=0.63.3',
-  'pyproject-metadata>=0.6.1', # not a hard dependency, only needed for projects that use PEP 621 metadata
```

Comment:
agreed with removing this, it just raises a "why is this here and why do I need to care" question mark.